Nest/00. Plan.md

```diff
@@ -0,0 +1,58 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
```

Comment:
Maybe this title could be made a bit simpler ?
IMHO, an end user would rather think about "Updating the list of available packages" rather than "Synchronizing the local cache", so that would make this item easier to find for him/her.

Reply:
To combat this sort of problem, you could have quick "also know as" section below titles, although tbh I've never seen that in other user guides / specifications.

Reply:
Maybe I could write in the chapter that it's about synchronizing the local cache, but not in the plan then